app/main.py

```
@@ -213,6 +214,44 @@ async def bot_connect(
+                    ),
+                },
+            )
+            if conversation:
+                logger.bind(session_id=session_id).info(
+                    f"Conversation record created for session {session_id}"
+                )
+            else:
+                logger.error(
+                    f"Failed to create conversation record for session {session_id}"
+                )
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)